service/src/test/resources/application-test.properties

```diff
@@ -1,3 +1,3 @@
 spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
 spring.datasource.url=jdbc:tc:postgresql://localhost/consent
-
+sandbox.scaapproach=redirect
```

Comment:
We should be able to get this value from `ProfileService` without having to duplicate this value. Add at least a TODO here. If we need it in both contexts we have to refactor a bit.